src/array.js

```diff
@@ -164,7 +164,10 @@ inherits(ArraySchema, MixedSchema, {
 
   ensure() {
     return this.default(() => []).transform(
-      val => (val === null ? [] : [].concat(val)),
+      // We transform the original value to allow a single element to
+      // get wrapped in an array. The transform added in the Array schema’s
+      // constructor turns non-array, non-JSON-string values into null.
+      (val, orig) => (orig === null ? [] : [].concat(orig)),
```

Comment:
I think it may make sense to use the original value only if `value` is null, so you don't run into issues with values like `'[1,2,4]'`

Reply:
@fionawhim I just ran into a need for this fix. I agree with @jquense. I'd think most transforms would want this behavior to keep them chain-able... Can we get this implemented? Thanks!

Something like this:
```
(val, orig) => {
  const value = val || orig

  return (value === null ? [] : [].concat(value))
}